packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -514,7 +514,6 @@ - (void)captureOutput:(AVCaptureOutput *)output
       return;
     }
 
-    CFRetain(sampleBuffer);
```

Comment:
Is the bug that this is retained but not released before the `if(_audioIsDisconnected)` early returns?

Reply:
(I think you're right it doesn't need to be retained at all since `sampleBuffer` isn't used outside the scope of this method)
>     Clients that need to reference the CMSampleBuffer object outside of the scope of this method must CFRetain it and then CFRelease it when they are finished with it.